test/specs/collections/Breadcrumb/Breadcrumb-test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Breadcrumb from 'src/collections/Breadcrumb/Breadcrumb'
+import * as common from 'test/specs/commonTests'
+
+describe('Breadcrumb', () => {
+  common.isConformant(Breadcrumb)
+  common.hasUIClassName(Breadcrumb)
+  common.rendersChildren(Breadcrumb)
+
+  it('renders a <div /> element', () => {
+    shallow(<Breadcrumb />)
+      .should.have.tagName('div')
+  })
+
+  const sections = [
+    { text: 'Home', link: true },
+    { text: 'T-Shirt', href: 'google.com' },
+  ]
+
+  it('renders children with `sections` prop', () => {
+    const wrapper = shallow(<Breadcrumb sections={sections} />)
+
+    wrapper.find('BreadcrumbDivider').should.have.length(1)
+    wrapper.find('BreadcrumbSection').should.have.length(2)
```

Comment:
As noted in [#321 (comment)](https://github.com/TechnologyAdvice/stardust/pull/321/files#r70023249) let's use `chai-enzyme` assertions for much better assertions and error messages:

```diff
-wrapper.find('BreadcrumbDivider').should.have.length(1)
+wrapper.should.have.exactly(1).descendants('BreadcrumbDivider')
```
```diff
-wrapper.find('BreadcrumbSection').should.have.length(2)
+wrapper.should.have.exactly(2).descendants('BreadcrumbSection')
```

Compare the difference in output between `should.have.length(x)` and `should.have.exactly(x).descendants(...)`:

```jsx
shallow(<Textarea />)
  .find('textarea').should.have.length(2)

//=> expected { Object (component, root, ...) } to have a length of 2 but got 1
```
```jsx
shallow(<Textarea />)
  .should.have.have.exactly(2).descendants('textarea')

//=> expected <Textarea /> to have 2 descendants 'textarea' but actually found 1

         HTML:

         <textarea data-reactroot=""></textarea>
```